test/function/getter.unit

```diff
@@ -0,0 +1,6 @@
+40 columns                              |
+>>> Split before `get`.
+VeryLongTypeAnnotation get veryLongGetter => null;
+<<<
+VeryLongTypeAnnotation
+get veryLongGetter => null;
```

Comment:
Does this getter not have any indentation when split?

Reply:
This is a style change I made in the prototype based on what I saw in a few places in the Flutter repo.

It's really rare, but not impossible, for a field or variable declaration to split after the type annotation, or for a function declaration to split after the return type. It mostly happens when you have a really long generic type. When that happens, it's not really clear what the indentation should look like.

Many years ago, I basically just picked when I first implemented it, and decided it would indent the member, like:

```dart
VeryLongTypeAnnotation
    get veryLongGetter => null;
```

I don't think I ever particularly liked this style, and I don't think many people ran into it. When I was tweaking style rules during the prototype, I tried just not indenting there, and I think it looks better, especially in the context of a long parameter list:

```dart
function(
  SomeReallyLongParameterType
  aParameter,
  int anotherParameter,
) {
  ...
}
```

It's definitely up for discussion, though, if you think it looks weird.

Reply:
I don't have a very strong opinion about this. I just thought it looked different from how I expected a member to look, that's all.

If it's what we've had for a while, let's keep it as it is.

Reply:
I think we should expect some pushback if folks do notice it. It makes the second line look like it could return `dynamic`, but I do agree that it looks nicer, especially looking at the first line of the body against the signature line(s) above.